README.md

```diff
@@ -23,9 +23,12 @@ You can either install kube-bench through a dedicated container, or compile it f
 Run ```docker run --rm -v `pwd`:/host aquasec/kube-bench:latest```. This will copy the kube-bench binary and configuration to you host. You can then run ```./kube-bench <master|node>```.
 
 2. Install from sources:
-If Go is installed on the target machines, you can simply clone this repository and run as follows: 
+If Go is installed on the target machines, you can simply clone this repository and run as follows (assuming your [$GOPATH is set](https://github.com/golang/go/wiki/GOPATH)):
 ```go get github.com/aquasecurity/kube-bench```
-```cp $GOROOT/bin/kube-bench .```
+```go get https://github.com/masterminds/glide```
```

Comment:
Please fix the following:
1. Running the command in the command to install glide fails with:
```
package https:/github.com/masterminds/glide: "https://" not allowed in import path
```

2. Changing the command to 'go get github.com/masterminds/glide' fails with:
```
../../masterminds/glide/glide.go:31:2: case-insensitive import collision: "github.com/masterminds/glide/vendor/github.com/codegangsta/cli" and "github.com/Masterminds/glide/vendor/github.com/codegangsta/cli"
```